packages/generator/src/generators/app-generator.ts

```diff
@@ -171,8 +174,9 @@ export class AppGenerator extends Generator<AppGeneratorOptions> {
         ),
       )
     }
-
-    if (gitInitResult.status === 0) {
+    if (gitInitResult === undefined) {
+      return
+    } else if (gitInitResult.status === 0) {
       this.commitChanges()
     } else {
       log.warning("Failed to run git init.")
```

Comment:
Shouldn‘t this be printed immediately after it failed? After Line 51 for Example. This could also make this part of the code easier to read.

Reply:
Oh, didn't think about putting it down there. Good point, will def. make it more easily readable.